app/api/api/views.py

```diff
@@ -1266,18 +1220,6 @@ def get_serializer_class(self):
         return super().get_serializer_class()
 
 
-class ScanReportFilterViewSet(viewsets.ModelViewSet):
```

Comment:
Have these all been superceded elsewhere?

Reply:
Yes they are no longer in use.